services/content/contentserver/contentserver.go

```diff
@@ -101,12 +104,7 @@ func (s *service) List(req *api.ListContentRequest, session api.Content_ListServ
 	)
 
 	if err := s.store.Walk(session.Context(), func(info content.Info) error {
-		buffer = append(buffer, &api.Info{
-			Digest:    info.Digest.String(),
-			Size:      info.Size,
-			CreatedAt: protobuf.ToTimestamp(info.CreatedAt),
-			Labels:    info.Labels,
-		})
+		buffer = append(buffer, infoToGRPC(info))
```

Comment:
Previously only `CreatedAt` is set. With `infoToGRPC`, both `CreatedAt` and `UpdatedAt` will be set.

Reply:
Is the `UpdatedAt` field being used some where?